common:
#RS LiDar
    lid_topic: "/livox/lidar"
    imu_topic: "/livox/imu"
    gps_topic:  "/gnss" # czy
    # lid_topic: "/rslidar_points1"
    # imu_topic: "/imu/data1"
    time_sync_en: false         # ONLY turn on when external time synchronization is really not possible
    
preprocess:
    lidar_type: 1                # 1 for Livox serials LiDAR, 2 for Velodyne LiDAR, 3 for ouster LiDAR, 4 RS
    scan_line: 16
    scan_rate: 10                # only need to be set for velodyne, unit: Hz,
    blind: 0.1


    intensity_dient_min: 20 
    intensity_dient_max: 20 
    lookfor_min_num: 1
    lookfor_max_num: 1
    range_blind: 0.1 
    delete_fov_degree: 0 

    fov_degree: 360 
    vertical_degree: 80 
    vertical_degree_min: -40 
    vertical_degree_max: 40 
    image_fov_resolution: 0.2
    image_vertical_rate: 3 

    intensity_min_range: 1

mapping:
    acc_cov: 0.1
    gyr_cov: 0.1
    b_acc_cov: 0.0001
    b_gyr_cov: 0.0001
    fov_degree:    180
    image_gradient_threshold:  20
    det_range:     100.0
    extrinsic_est_en:  true      # true: enable the online estimation of IMU-LiDAR extrinsic,
    extrinsic_T: [ 0.25, 0.06, 0.65]
    extrinsic_R: [ 1, 0, 0,
                   0, 1, 0,
                   0, 0, 1]
    is_our_weight: true #是否使用自定义的权重，而非一样的权重 ， true表示使用自定义的权重 czy
    is_intensity_edge_windows: false  # 是否使用强度线点窗口 czy

publish:
    path_en:  true
    scan_publish_en:  true       # false: close all the point cloud output
    dense_publish_en: true       # false: low down the points number in a global-frame point clouds scan.
    scan_bodyframe_pub_en: true  # true: output the point cloud scans in IMU-body-frame
    pub_gps_path: false  # 是否有gps czy
    pub_odom_path: true # 是否保存位姿 czy

pcd_save:
    pcd_save_en: true
    interval: -1                 # how many LiDAR frames saved in each pcd file; 